app/constants/action-icons.js

```diff
@@ -0,0 +1,34 @@
+export const ACTIONS_ICONS = [
```

Comment:
This seems to be a duplicate of `constants/item-actions`

Also, `Edited`, `Inventory` and `Pack` shouldn't be in the bottom menu.

`Edited` isn't a valid inventory action at all

Reply:
Can you point out what the actual change is ?

Reply:
Also note that the title of the PR is "fix icons on history page", but it's modifying a helper that can be used _everywhere_

Reply:
`constants/item-actions` this is specifically used in the actions dropdown next to designate button on item-published tab. And those are specific to item gain-loss.

On history tab, the actions like `pack`, `Edited`(groups item attribute updates), `inventory` are listed which are missing in  `constants/item-actions` and hence added different files only specific for icons.

Reply:
Aaah, gotcha ! I thought these were for the bottom menu. Looks good then :)